firestore/src/android/firestore_android.h

```diff
@@ -214,6 +214,8 @@ class FirestoreInternal {
 #else   //  defined(_STLPORT_VERSION)
   std::unordered_set<ListenerRegistrationInternal*> listener_registrations_;
 #endif  //  defined(_STLPORT_VERSION)
+
+  Mutex bundle_listener_mutex_;
```

Comment:
ultranit: The mutex should be named `bundle_listeners_mutex_`, instead of `bundle_listener_mutex_`.